src/nxt_file.c

```diff
@@ -312,7 +312,7 @@ nxt_file_rename(nxt_file_name_t *old_name, nxt_file_name_t *new_name)
  * Linux 2.6 does not use BKL.
```

Comment:
The comment here needs to be updated.

Reply:
What do you mean? Document the behavior on newer kernels?

Reply:
I meant that we should add to this comment a line about why `&& !(NXT_SOLARIS)` has been added.

Reply:
I updated, take a look.